manifest.json.template

```diff
@@ -12,68 +12,79 @@
   },
   "binaryDependencies": {
     "org.zowe.zlux.zlux-core": {
-      "version": "~1.7.0-STAGING",
+      "version": "~1.8.0-STAGING",
```

Comment:
It looks like you've not reverted the manifest values to point to the correct staging branches